app/controllers/sufia/admin/admin_sets_controller.rb

```diff
@@ -95,7 +95,7 @@ def list_search_builder
       end
 
       def create_admin_set
-        AdminSetCreateService.new(@admin_set, current_user).create
+        AdminSetCreateService.new(@admin_set, current_user, selected_workflow).create
```

Comment:
That's the nature of Emacs, https://chocolatey.org/content/packageimages/vim.8.0.568.svg would never do that.